api/utils/marshamallow_schemas.py

```diff
@@ -0,0 +1,103 @@
+"""Validation schemas."""
+from marshmallow import (Schema, fields, post_load, validate,
+                         ValidationError)
+from api.models import User, Activity, ActivityType
+
+
+class BaseSchema(Schema):
+    """Creates a base validation schema."""
+
+    uuid = fields.String(dump_only=True, validate=[
+        validate.Length(max=36)])
+    name = fields.String(
+        required=True,
+        error_messages={
+            'required': {'message': 'A name is required.',
+                         'status_code': 400}
+        })
+    photo = fields.String()
+    created_at = fields.DateTime(dump_only=True)
+    modified_at = fields.DateTime(dump_only=True)
+    description = fields.String()
+
+    @post_load
+    def verify_activity_type(self, data):
```

Comment:
This method shouldn't be in the `BaseSchema`, right?

Reply:
@LarryWachira Fixed. I have moved it to the `Activity Schema`